kernels/volk/volk_32f_s32f_clamppuppet_32f.h

```diff
@@ -0,0 +1,65 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2023 Magnus Lundmark <magnuslundmark@gmail.com>
+ *
+ * This file is part of VOLK
+ *
+ * SPDX-License-Identifier: LGPL-3.0-or-later
+ */
+
+#ifndef INCLUDED_volk_32f_s32f_clamppuppet_32f_H
+#define INCLUDED_volk_32f_s32f_clamppuppet_32f_H
+
+#include <volk/volk_32f_s32f_x2_clamp_32f.h>
+
+#ifdef LV_HAVE_AVX2
+static inline void volk_32f_s32f_clamppuppet_32f_a_avx2(float* out,
+                                                        const float* in,
+                                                        const float min,
+                                                        unsigned int num_points)
+{
+    volk_32f_s32f_x2_clamp_32f_a_avx2(out, in, min, .5f, num_points);
```

Comment:
The `min` value is set to `-0.5` in `kernel_tests.h`. Would it make sense to set `max = -1.0 * min` or smth like `max = -2.0 * min` to test this kind of parameters?

Reply:
Sure, makes it more convenient to change the parameter as well.